src/sentry/search/events/builder.py

```diff
@@ -147,7 +200,9 @@ def __init__(
     ):
         self.dataset = dataset
 
-        self.params = params
+        # filter params is the older style params, shouldn't be used anymore
+        self.filter_params = params
```

Comment:
Still used in `arguments = snql_function.format_as_arguments(...` will remove its usage their in a follow up PR